lightning/src/ln/channel.rs

```diff
@@ -6413,6 +6311,113 @@ impl<SP: Deref> OutboundV1Channel<SP> where SP::Target: SignerProvider {
 
 		Ok(())
 	}
+
+	/// Handles a funding_signed message from the remote end.
+	/// If this call is successful, broadcast the funding transaction (and not before!)
+	pub fn funding_signed<L: Deref>(
```

Comment:
Ok this makes sense for unfunded V1 channels since there's no need to persist the channel before a `funding_signed`. Also it's great that this is only in outbound now.
It has given me a little more to think about in the V2 case.